README.md

```diff
@@ -945,7 +945,7 @@ DeviceInfo.getPreviewSdkInt().then(previewSdkInt => {
 
 ### getReadableVersion()
 
-Gets the application human readable version (same as getVersion() + '.' + getBuildCode())
+Gets the application human readable version (same as getVersion() + '.' + getBuildNumber())
```

Comment:
This is correct! Thanks for the correction 
https://github.com/react-native-community/react-native-device-info/blob/master/src/index.ts#L322